README.md

```diff
@@ -1,32 +1,50 @@
 # Bino
 
 The booking bot at your service
+### Requirements
 
-### Configure Database 
+- [Go](https://golang.org/) 1.16
+- [Docker](https://www.docker.com/)
 
-Copy file `.env-sample` to `.env.docker`
+### Configure Database
+
+Both `MYSQL_ROOT_PASSWORD` and `MYSQL_DATABASE` are required to run the database. They are made available to the database via a file named `.env.docker`.
+
+For a quick setup you can copy `.env-sample`:
 
 ```shell
-$ cp .env-sample .env.docker
+cp .env-sample .env.docker
 ```
+### Run the bino bot
 
-Run the command for import dump in `mysql`
+#### Setup Discord Token
+
+Bino needs to connect to a Discord server to run, so you'll need to provide a bot token. Check [WriteBots](https://writebots.com) great [tutorial](https://www.writebots.com/discord-bot-token/) to setup one.
+
+The token needs to be available via an environment variable named `BINO_DISCORD_TOKEN`. You can do it by running:
 
 ```shell
-$ docker exec -i binodb mysql -uroot -pdocker bino < ./db/Bino.sql
+export BINO_DISCORD_TOKEN=<your_token_goes_here>
 ```
 
-### Run the bino bot
+#### Install dependencies
 
-Execute this command for create container:
+It can be done via:
 
 ```shell
-$ docker-compose run --service-ports --rm app bash
```

Comment:
this command is required to enter the container.

Reply:
Would you mind elaborating a bit of the intent of this command? I don't think I fully get it.

If the goal is to run the app I guess `docker-compose up` does it all by bringing up the services: app and db and making sure they are happy.

On the other hand I can see `run` being helpful while developing, you could start and stop the app and see the changes while still still the container.

Reply:
Yes! For development this is necessary

Reply:
Ok, so I've cleaned up the `docker-compose.yaml` file and added a `Dockerfile` - so the dependencies are already downloaded when you run the container.

Hopefully this would make it easier to use docker while developing.

Updated the README to reflect changes.